include/mme-app/utils/mmeProcedureTypes.h

```diff
@@ -89,6 +90,20 @@ namespace mme
  	intraMmeS1Ho_c
  };
 
+ struct BearerStatusCompareById
```

Comment:
is this correct file ?

Reply:
Yes, as this is being used by MmeSmCreateBearerProcCtxt, we placed it here.